Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -84,9 +88,48 @@ the web document root - a situation that is not optimal from a security point of
 .. _Environment-current-script:
 
 getCurrentScript()
-==================
+------------------
+
+Returns the path and filename to the current PHP script.
+
+Configuring Environment Paths
+=============================
+
+The environment variable called :php:`TYPO3_PATH_APP` is used
+to allow to store data outside of the document root.
+
+All composer-based installations benefit from this functionality, as data that was previously
+stored and hard-coded within :file:`typo3temp/var/` is now stored within the **project root** folder :file:`var/`.
+
+For non-composer installations (Classic Mode), it is possible to set the environment variable to a folder, usually one level
+upwards than the regular **webroot**. This increases security for any TYPO3 installation as files are not
+publicly accessible (for example via web browser) anymore.
+
+A typical example:
+
+- :php:`TYPO3_PATH_APP` is set to :file:`/var/www/my-project`.
+- The web folder is then set to :php:`TYPO3_PATH_ROOT` :file:`/var/www/my-project/public`.
```

Comment:
Since `TYPO3_PATH_ROOT` shows up rather sudden here I think we should mention this in the first paragraph already and make the difference to `TYPO3_PATH_APP` crystal clear.

Reply:
Can you check my explanation? I'm not quite sure about `crystal clear` ;)

Reply:
I think it works like this. I just wanted to be sure both are mentioned since I'm speaking from personal experience. ;-)